### PR TITLE
[Build Pipeline] Add ability to skip testing

### PR DESCRIPTION
**Pull request by jashook**
/cc @RussKeldorph

### CODE REVIEW

File: buildpipeline/Dotnet-CoreClr-Trusted-BuildTests.json [hunk 1]
```diff
@@ -203,6 +210,10 @@
       "value": "$(Build.BuildNumber)",
       "allowOverride": true
     },
+    "PB_SkipTests": {
+      "value": "false",
+      "allowOverride": false
```
Comment (RussKeldorph) [inline buildpipeline/Dotnet-CoreClr-Trusted-BuildTests.json]: Is 'false' intentional?  Temporary until something else happens?

Comment (jashook) [reply buildpipeline/Dotnet-CoreClr-Trusted-BuildTests.json]: No typo. Thank you for catching

**Comment by jashook** [comment]
Addressed

**Comment by jashook** [comment]
Ping @dotnet/microsoft

**Comment by jashook** [comment]
Ping @mmitche

**Pull request merged**